app/models/user.rb

```diff
@@ -561,15 +559,7 @@ def any_kind_of_delegate?
   end
 
   def trainee_delegate?
-    delegate_status == "trainee_delegate"
-  end
-
-  def staff_delegate?
```

Comment:
Where did this method go?

Reply:
This method is not used anywhere, hence removed it.

Reply:
Are you sure? I feel like `competition.rb` has some "make sure that Delegates for this competition are actually Delegates" validation :thinking:

Reply:
Okay, gotcha